astropy/wcs/src/wcslib_wtbarr_wrap.c

```diff
@@ -0,0 +1,238 @@
+/*
+ Author: Mihai Cara
```

Comment:
We generally don't add authorship on a per file basis. Would you mind removing this?